task/dockerslim-build/0.1/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+reviewers:
```

Comment:
can we omit this file if nobody is the owner of this task?

Reply:
How does it work? Isn't this file supposed to list the usernames for organization members that did review / approve that task folder getting merged?
For sure, I can remove that one, if it's not mandatory.